src/runtime_src/core/tools/common/ReportDebugIpStatus.cpp

```diff
@@ -495,12 +493,14 @@ DebugIpStatusCollector::readAccelDeadlockDetector(debug_ip_data* dbgIpInfo)
 }
 
 
-void 
+bool 
```

Comment:
Why the introduction of a boolean return type.

I ask for when I look at the code, it "seems" that this value is used only do indicate that the `total_num_debug_ips` value is non-zero.  Instead of doing "double duty" here, it would be better if this count is extracted from the returned property tree, like you are doing else where in the code:
`uint64_t num_debug_ips = _dbgIpStatus_pt.get<uint64_t>("total_num_debug_ips");
`